src/Stratis.Bitcoin.Features.MemoryPool.Tests/MempoolPersistenceTest.cs

```diff
@@ -236,7 +236,7 @@ public void SaveStreamTest()
 
         private NodeSettings CreateSettings(string subDirName)
         {
-            return new NodeSettings(args: new string[] { $"-datadir={ Directory.CreateDirectory(Path.Combine(this.dir, subDirName)).FullName }" });
+            return new NodeSettings(KnownNetworks.TestNet, args: new string[] { $"-datadir={ Directory.CreateDirectory(Path.Combine(this.dir, subDirName)).FullName }" });
```

Comment:
should probably use `this.network` here, like the other tests (it's mainnet, incidentally).

Reply:
good spot